### ISSUE TITLE
Fetching a retweet doesn't retrieve its full text

### ISSUE THREAD

**Issue by Jeto143** [Rishikant181/Rettiwt-API#521]
Using version 2.6.2.

```js
const tweet = await rettiwt.tweet.details('1777668663400661199');
console.log(tweet.fullText);
```

will log:

> RT @LaravelPodcast: 🌟 New episode alert! 🌟  Join us on the Laravel podcast as we talk about the success of Laravel 11's release, the introd…

As you can see, the text gets truncated, while the [actual retweet](https://twitter.com/taylorotwell/status/1777668663400661199) is complete.

The same occurs when retrieving the tweet from other methods, such as a user's timeline.

Note: there also doesn't seem to be any way to retrieve the ID of the original tweet, which would be an okay workaround.

Potentially relevant discussion (although this may be for another API version):
https://devcommunity.x.com/t/twitter-api-v2-only-retweet-text-truncated/151128

Potentially related issue (although it's probably a different one):
https://github.com/Rishikant181/Rettiwt-API/issues/471

**Comment by Rishikant181** [comment]
On it

**Comment by Rishikant181** [comment]
@Jeto143 Okay so the returned tweet now includes the original tweet which was retweeted in a field called `retweetedTweet`. Please update to v2.7.0 to see the changes.

**Comment by Rishikant181** [comment]
Closed #522
